service/worker/migration/force_replication_workflow.go

```diff
@@ -37,17 +38,28 @@ import (
 type (
 	ForceReplicationParams struct {
 		Namespace               string
-		Query                   string // query to list workflows for replication
+		Query                   string   // DEPRECATED: use Queries
```

Comment:
Since the workflow code is not compatible anyway, I would suggest we remove this entirely.

Reply:
i would prefer keeping this var for 1 to 2 releases for compatibility (so caller of this workflow can have the time to update)